src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/support/SupportService.kt

```diff
@@ -6,23 +6,26 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.HdcService
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.IS91DeterminationService
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerSearchApiClient
 import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerSearchPrisoner
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.DeliusApiClient
 
 @Service
 class SupportService(
   private val prisonerSearchApiClient: PrisonerSearchApiClient,
   private val hdcService: HdcService,
   private val eligibilityService: EligibilityService,
   private val iS91DeterminationService: IS91DeterminationService,
+  private val deliusApiClient: DeliusApiClient,
 ) {
   fun getIneligibilityReasons(prisonNumber: String): List<String> {
     val prisoners = prisonerSearchApiClient.searchPrisonersByNomisIds(listOf(prisonNumber))
     if (prisoners.size != 1) {
       error("Found ${prisoners.size} prisoners for: $prisonNumber")
     }
     val prisoner = prisoners.first()
-    val reasons = eligibilityService.getIneligibilityReasons(prisoner)
+    val deliusRecord = deliusApiClient.getOffenderManagers(listOf(prisoner.prisonerNumber)).first()
+    val eligibility = eligibilityService.getEligibilityAssessment(prisoner, deliusRecord.team.provider.code)
```

Comment:
nitpick: is eligibility an eligibilityAssessment

Reply:
It is, I've battled with myself a little on this, as `eligibilityAssessment` is quite long to write out everywhere, but neither `eligibility` or `assessment` are especially accurate as a shorthand 😅